### PR TITLE
fix: allow to mark http url as trusted

### PR DESCRIPTION
**Pull request by ritikverma2000**
I have added http inside trustedSource.startsWith .

image

### CODE REVIEW

File: src/main/java/dev/jbang/net/TrustedSources.java [hunk 1]
```diff
@@ -85,7 +85,7 @@ public boolean isURLTrusted(URI url) throws URISyntaxException {
 			}
 
 			URI parsedTrustedSource;
-			if (trustedSource.startsWith("https://")) {
+			if (trustedSource.startsWith("http://")) {
```
Comment (maxandersen) [inline src/main/java/dev/jbang/net/TrustedSources.java:88]: Now it won't work with https so that does not work as fix.

**Review by maxandersen — CHANGES_REQUESTED** [review]

File: src/main/java/dev/jbang/net/TrustedSources.java (hunk 1 above)
Comment (ritikverma2000) [reply src/main/java/dev/jbang/net/TrustedSources.java:88]: Okay I can just add a or condition in here then @maxandersen what do you suggest.

Comment (maxandersen) [reply src/main/java/dev/jbang/net/TrustedSources.java]: Yes. That seems reasonable. 

I highly expect the tests to fail with the current suggestion so I suggest you run the tests, then see what fails and add a test that test for http based URL. Then fix it so all the tests passes.

Comment (ritikverma2000) [reply src/main/java/dev/jbang/net/TrustedSources.java]: @maxandersen we have this method already present. 
image

Comment (maxandersen) [reply src/main/java/dev/jbang/net/TrustedSources.java]: Yes. So add a test that uses http so we can verify it failed before and fixed now with the change.

**Comment by maxandersen** [comment]
The build is failing because formatting not applied and no tests added to verify this actually works. 

Please run `gradle spotlessApply build clean` to have formatting applied and have tests pass. See CONTRIBUTING.MD for details.


Thanks.

**Comment by ritikverma2000** [comment]
> The build is failing because formatting not applied and no tests added to verify this actually works.
> 
> Please run `gradle spotlessApply build clean` to have formatting applied and have tests pass. See CONTRIBUTING.MD for details.
> 
> Thanks.

Sure @maxandersen

**Comment by maxandersen** [comment]
Thans @ritikverma2000 I fixed the formatting so builds will work.

**Pull request merged**